packages/muelu/doc/UsersGuide/masterList.xml

```diff
@@ -386,6 +386,15 @@
       <comment-ML>parameter not existing in ML</comment-ML>
     </parameter>
 
+    <parameter>
+      <name>aggregation: distance laplacian algo</name>
+      <type>string</type>
+      <default>"default"</default>
+      <description>Type of sub-algorithm for "distance laplacian" dropping. Possible values: "default", "unscaled cut", "scaled cut".</description>
+      <visible>true</visible>
```

Comment:
Make the visibility `false` so that it doesn't show up in the manual.

Reply:
That was the intent :-/
Good catch!